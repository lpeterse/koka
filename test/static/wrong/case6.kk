// case sensitivity
import moda = static/modA
import static/modB

alias wrong = moda/ainT
