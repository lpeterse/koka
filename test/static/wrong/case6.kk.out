test/static/wrong/case6.kk(5,15): error: type ainT should be cased as aint